docs/admin/install.rst

```diff
@@ -41,8 +41,6 @@ translation-finder
     https://github.com/WeblateOrg/translation-finder
 Python Social Auth
     https://python-social-auth.readthedocs.io/
-Whoosh
-    https://bitbucket.org/mchaput/whoosh/wiki/Home
```

Comment:
Whoosh is still used for query parsing, but its replacement in this case is planned as well, see https://github.com/WeblateOrg/weblate/issues/3229. But it's probably okay to remove it from this list now.

Reply:
Oh, good to know! I read in the changelog that it was replaced by DB FTS for searching, but I was definitely grepping one directory level above where I should 😅